lib/iris/tests/integration/netcdf/test__chunk_control.py

```diff
@@ -0,0 +1,161 @@
+# Copyright Iris contributors
+#
+# This file is part of Iris and is released under the LGPL license.
+# See COPYING and COPYING.LESSER in the root of the repository for full
+# licensing details.
+"""Integration tests for loading and saving netcdf files."""
```

Comment:
This file is really filling the unit test role. I reckon move these into a section in [`unit/fileformats/netcdf/loader/test_load_cubes.py`](https://github.com/SciTools/iris/blob/80c179251219eebdfd03e72b0df123c2cfca61fd/lib/iris/tests/unit/fileformats/netcdf/loader/test_load_cubes.py)

Or maybe a dedicated module called `test_load_cubes_chunk_control.py`?

I don't think there are any true integration tests that are worth adding for this feature.